config/environments/production.rb

```diff
@@ -81,12 +81,6 @@
   # Send deprecation notices to registered listeners.
```

Comment:
line 58, the new log level is `:info` by default https://github.com/rails/rails/pull/39707